lib/compat/wordpress-6.8/class-gutenberg-rest-comment-controller-6-8.php

```diff
@@ -42,6 +42,11 @@ public function create_item_permissions_check( $request ) {
 			}
 		}
 
+		// Skip all the checks if comment type is not comment.
+		if( 'comment' !== $request['comment_type'] ) {
+			return true;
```

Comment:
Do we not still need the checks for a valid post ID, trash status, and read permission?

Reply:
What I would do to make things clearer: if the comment type is `comment`, run the parent `create_item_permissions_check` method so we don't need to implement it here. For all other types, re-add the checks that we do want to keep, which I think would be: check if the user is logged in, the post ID is valid, it's not in the trash, and the user can read the post.